index.html

```diff
@@ -3532,97 +3532,30 @@ <h3>Terms of Use</h3>
     "dateOfBirth": "1930-10-01"
   },
   "credentialSchema": {
-    "id": "https://api-test.ebsi.eu/trusted-schemas-registry/v2/schemas/z3MgUFUkb722uq4x3dv5yAJmnNmzDFeK5UC8x83QoeLJM",
+    "id": "https://code.europa.eu/ebsi/json-schema/-/raw/db67c771b96715c23ddf384abcfef33d6174d494/schemas/vcdm2.0/trust-model/schema.json",
     "type": "JsonSchema"
   },
   "termsOfUse": {
-    "id": "https://api-test.ebsi.eu/trusted-issuers-registry/v4/issuers/did:ebsi:zz7XsC9ixAXuZecoD9sZEM1/attributes/7201d95fef05f72667f5454c2192da2aa30d9e052eeddea7651b47718d6f31b0",
-    "type": "IssuanceCertificate"
+    "type": "TrustFrameworkPolicy",
+    "trustFramework": "EBSI_TRUST_FRAMEWORK_URI_GOES_HERE",
+    "policyId": "EBSI_POLICY_ID_URI_GOES_HERE"
   }
 }
         </pre>
 
         <p>
-In the example above, the [=issuer=] is asserting that as a European
```

Comment:
Please do not remove lines 3546 to 3552 as this description is still approximately correct